tdms/src/interpolate.cpp

```diff
@@ -67,71 +103,48 @@ void interpolateFieldCentralE( double ***Ex_yee, double ***Ey_yee, double ***Ez_
+	      }
 }
-/*Interpolate the electric field to the origin of the Yee cell
- *
- *Ex_yee[in] - steady state x component of electric field calculated at points in the Yee cell
- *Ey_yee[in] - steady state y component of electric field calculated at points in the Yee cell
- *Ez_yee[in] - steady state z component of electric field calculated at points in the Yee cell
- *
- *Ex[out] - steady state x component of electric field interpolated to Yee cell origin
- *Ey[out] - steady state y component of electric field interpolated to Yee cell origin
- *Ez[out] - steady state z component of electric field interpolated to Yee cell origin
- *
- *I[in] - number of elements in the i direction of the FDTD grid
- *J[in] - number of elements in the j direction of the FDTD grid
- *K[in] - number of elements in the k direction of the FDTD grid
+
+/**
+ * @brief Interpolate the TE electric field (x,y components) to the origin of the Yee cell
  *
- *i_l[in]  - lowest i index into the FDTD grid to evaluate the field at. Should be >= 2.
- *i_u[out] - largest i index into the FDTD grid to evaluate the field at. Should be <= I-2.
- *j_l[in]  - lowest i index into the FDTD grid to evaluate the field at. Should be >= 2.
- *j_u[out] - largest i index into the FDTD grid to evaluate the field at. Should be <= J-2.
- *k_l[in]  - lowest i index into the FDTD grid to evaluate the field at. Should be >= 2.
- *k_u[out] - largest i index into the FDTD grid to evaluate the field at. Should be <= K-2.
+ * @param[in] Ex_yee Steady state x component of electric field calculated at points in the Yee cell
+ * @param[in] Ey_yee Steady state y component of electric field calculated at points in the Yee cell
+ * @param[in] Ez_yee Steady state z component of electric field calculated at points in the Yee cell
+ * @param[out] Ex Steady state x component of electric field interpolated to Yee cell origin
+ * @param[out] Ey Steady state y component of electric field interpolated to Yee cell origin
+ * @param[out] Ez Steady state z component of electric field interpolated to Yee cell origin
+ * @param[in] I Number of elements in the i direction of the FDTD grid
+ * @param[in] J Number of elements in the j direction of the FDTD grid
+ * @param[in] K Number of elements in the k direction of the FDTD grid
+ * @param[in] i_l Least i index into the FDTD grid to evaluate the field at. Should be >= 2
+ * @param[in] i_u Greatest i index into the FDTD grid to evaluate the field at. Should be <= I-2
+ * @param[in] j_l Least j index into the FDTD grid to evaluate the field at. Should be >= 2
+ * @param[in] j_u Greatest j index into the FDTD grid to evaluate the field at. Should be <= J-2
+ * @param[in] k_l Least k index into the FDTD grid to evaluate the field at. Should be >= 2
+ * @param[in] k_u Greatest k index into the FDTD grid to evaluate the field at. Should be <= K-2
  */
 void interpolateFieldCentralE_TE( double ***Ex_yee, double ***Ey_yee, double ***Ez_yee,
```

Comment:
these can now just be deleted?!

Reply:
They're still called (for the time being) in `iterator.cpp`, but they can definitely go once I've tested `interpolatetimeDomainEcomponent` is doing what it should be